Robot_Adapter/CRUD/Read/Loads/Loadcombination.cs

```diff
@@ -0,0 +1,72 @@
+﻿/*
+ * This file is part of the Buildings and Habitats object Model (BHoM)
+ * Copyright (c) 2015 - 2019, the respective contributors. All rights reserved.
+ *
+ * Each contributor holds copyright over their respective contributions.
+ * The project versioning (Git) records all such contribution source information.
+ *                                           
+ *                                                                              
+ * The BHoM is free software: you can redistribute it and/or modify         
+ * it under the terms of the GNU Lesser General Public License as published by  
+ * the Free Software Foundation, either version 3.0 of the License, or          
+ * (at your option) any later version.                                          
+ *                                                                              
+ * The BHoM is distributed in the hope that it will be useful,              
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of               
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the                 
+ * GNU Lesser General Public License for more details.                          
+ *                                                                            
+ * You should have received a copy of the GNU Lesser General Public License     
+ * along with this code. If not, see <https://www.gnu.org/licenses/lgpl-3.0.html>.      
+ */
+
+using System.Collections.Generic;
+using RobotOM;
+using BH.oM.Structure.Loads;
+using System.Diagnostics;
+
+namespace BH.Adapter.Robot
+{
+    public partial class RobotAdapter
+    {
+        /***************************************************/
+        /****           Private Methods                 ****/
+        /***************************************************/
+
+        private List<LoadCombination> ReadLoadCombinations(List<string> ids = null)
+        {
+            m_RobotApplication.Project.Structure.Cases.BeginMultiOperation();
+            List<LoadCombination> bhomLoadCombinations = new List<LoadCombination>();
+            IRobotCaseCollection rLoadCollection = m_RobotApplication.Project.Structure.Cases.GetAll();
+
+            for (int i = 1; i <= rLoadCollection.Count; i++)
+            {
+                IRobotCase rCase = rLoadCollection.Get(i) as IRobotCase;
+                if (rCase.Type == IRobotCaseType.I_CT_COMBINATION) // No support for code combinations for now as it needs building support for components
+                {
+                    IRobotCaseCombination rCaseCombination = rCase as IRobotCaseCombination;
+                    List<System.Tuple<double, ICase>> bCaseFactors = new List<System.Tuple<double, ICase>>();
+
+                    for (int j = 1; j <= rCaseCombination.CaseFactors.Count; j++)
+                    {
+                        IRobotCase rCaseIn = rLoadCollection.Get(j) as IRobotCase;
```

Comment:
I don't think this returns the correct case corresponding to the case factor - see below suggested change. I ran a quick test on this and seemed to work - see before and after screenshots in the test script. 

`IRobotCase rCaseIn = rLoadCollection.Get(rCaseCombination.CaseFactors.Get(j).CaseNumber) as IRobotCase;`

![image](https://user-images.githubusercontent.com/15233608/74358975-4b409180-4dba-11ea-96fb-03d199956bbf.png)

![image](https://user-images.githubusercontent.com/15233608/74359140-9bb7ef00-4dba-11ea-8752-1b55b1d70ae8.png)

Reply:
That is correct, Robot expose loads in a load combo in the order they were defined, which can be different from their number. Your suggestion fixes this issue. Thanks for the code suggestion!